base_classes/NXcg_polyline.nxdl.xml

```diff
@@ -50,58 +50,36 @@ multiple vertices possible with the same point coordinates but different names.-
-    <!--number_of_unique_vertices(NX_POSINT):
-  unit: NX_UNITLESS-->
-    <field name="number_of_total_vertices" type="NX_POSINT" units="NX_UNITLESS">
+    <field name="depends_on" type="NX_CHAR">
         <doc>
-             The total number of vertices, irrespective of their eventual uniqueness,
-             i.e. the total number of vertices that have to be visited when walking
-             along each polyline.
+             Reference to an instance of :ref:`NXcg_point` which defines the
+             location of the vertices that are referred to in the
+             NXcg_polyline instance.
         </doc>
     </field>
-    <field name="number_of_vertices" type="NX_POSINT" units="NX_UNITLESS">
+    <field name="number_of_unique_vertices" type="NX_POSINT" units="NX_UNITLESS">
         <doc>
-             Array which specifies of how many vertices each polyline is built.
-             The number of vertices represent the total number of vertices for
-             each polyline, irrespectively whether vertices are shared or not.
-             See the docstring for polylines for further details about how
-             a set with different polyline members should be stored.
+             The total number of vertices that have different positions.
         </doc>
-        <dimensions rank="1">
-            <dim index="1" value="c"/>
-        </dimensions>
     </field>
-    <group type="NXtransformations">
-        <doc>
-             Reference to or definition of a coordinate system with
-             which the qualifiers and polyline data are interpretable.
-        </doc>
-    </group>
-    <field name="identifier_offset" type="NX_INT" units="NX_UNITLESS">
+    <field name="number_of_total_vertices" type="NX_INT" units="NX_UNITLESS">
         <doc>
-             Integer which specifies the first index to be used for distinguishing
-             polylines. Identifiers are defined either implicitly
-             or explicitly. For implicit indexing the identifiers are defined on the
-             interval [identifier_offset, identifier_offset+c-1].
-             For explicit indexing the identifier array has to be defined.
-             
-             The identifier_offset field can for example be used to communicate if the
-             identifiers are expected to start from 1 (referred to as Fortran-/Matlab-)
-             or from 0 (referred to as C-, Python-style index notation) respectively.
+             The total number of vertices, irrespective of their eventual uniqueness.
         </doc>
     </field>
-    <field name="identifier" type="NX_INT" units="NX_UNITLESS">
+    <field name="number_of_vertices" type="NX_INT" units="NX_UNITLESS">
         <doc>
-             Integer used to distinguish polylines for explicit indexing.
+             The total number of vertices of each polyline, irrespectively
+             whether vertices are shared by vertices or not.
+             See the docstring for polylines for further details about how
```

Comment:
Not sure if this last sentence needs to be stated here

Reply:
Addressed